WindowsFunctionsStacks.json

```diff
@@ -52,6 +52,37 @@
                 "isDeprecated": null
             }
         },
+        {
+            "id": null,
+            "name": ".Net Framework",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=WindowsFunctions",
+            "properties": {
+                "name": ".Net Framework",
+                "display": ".Net Framework",
+                "dependency": null,
+                "majorVersions": [
+                    {
+                        "displayVersion": "4.7",
```

Comment:
Sorry, can you explain why it's 4.7 here

Reply:
This is V1 app. .Net Framework version 4.7  https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions

Reply:
OK I see now